cmd/transcoding/transcoding.go

```diff
@@ -18,8 +18,9 @@ func validRenditions() []string {
 }
 
 func main() {
+	var err error
 	if len(os.Args) <= 3 {
-		panic("Usage: <input file> <output renditions, comma separated> <sw/nv>")
+		panic("Usage: <input file> <output renditions, comma separated> <sw/nv> <from> <to>")
```

Comment:
I think there's a missing argument here: 
```suggestion
		panic("Usage: <input file> <output renditions, comma separated> <sw/nv> [<gpu_id>] <from> <to>")
```
So I agree with @oscar-davids that using [flag](https://pkg.go.dev/flag) for parsing might be a good idea here, and have consistent argument names compared to the `livepeer` and `livepeer_bench` tools.